pkg/host/internal/network/network.go

```diff
@@ -95,10 +96,33 @@ func (n *network) TryGetInterfaceName(pciAddr string) string {
 		return name
 	}
 
-	log.Log.V(2).Info("tryGetInterfaceName()", "name", netDevName)
+	log.Log.V(2).Info("TryGetInterfaceName()", "name", netDevName)
 	return netDevName
 }
 
+// GetInterfaceIndex tries to find the SR-IOV virtual interface index base on pci address
```

Comment:
this function is not limited to SRIOV virtual interfaces as far as i can tell. 

how about:
`// GetInterfaceIndex returns network interface index base on pci address or error if occurred`